KubeArmor/eventAuditor/BPF/const-x64.go

```diff
@@ -0,0 +1,816 @@
+// x86 64bit syscall numbers
+// Also used as event IDs
+// https://github.com/torvalds/linux/blob/master/arch/x86/entry/syscalls/syscall_64.tbl
+
+// Define the syscall used with respect to x64 arch
+// +build amd64
+
+package main
```

Comment:
main? also, KubeArmor/eventAuditor/BPF directory is no longer available.
you need to update your code on the latest code